Jenkinsfile

```diff
@@ -1,16 +1,22 @@
-@Library('jenkins-pipeline-shared-libraries')_
+@Library('jenkins-pipeline-shared-libraries-ginxo')_
```

Comment:
@Ginxo REMINDER: before merging please remove this link pointingto libraries-ginxo